py-polars/polars/lazy/frame.py

```diff
@@ -388,6 +388,39 @@ def filter(self, predicate: "Expr") -> "LazyFrame":
         ----------
         predicate
             Expression that evaluates to a boolean Series.
+
+        Examples
+        --------
+
+        >>> df = pl.DataFrame({
+        >>>     "foo": [1, 2, 3],
+        >>>     "bar": [6, 7, 8],
+        >>>     "ham": ['a', 'b', 'c']
+        >>> }).lazy()
+        >>> # Filter on one condition
+        >>> df.filter(pl.col("foo") < 3).collect()
```

Comment:
Shall we name the variable differently? I always think of a `DataFrame` when reading `df`.

`LazyFrame -> lf` ?

Reply:
Yep, makes sense. I'll make the change